lib/data/landing_page_content_items/homepage.yaml

```diff
@@ -81,10 +81,12 @@ blocks:
     blocks:
       - type: heading
         content: Our tasks
-        path: /landing-page/tasks
       - type: govspeak
         content: |
           <p>Lorem ipsum dolor sit amet. In voluptas dolorum vel veniam nisi et voluptate dolores id voluptatem distinctio. Et quia accusantium At ducimus quis aut voluptates iusto aut esse suscipit.</p>
+      - type: govspeak
+        content: |
+          <p><a href="/landing-page/tasks" class="govuk-link govuk-!-font-weight-bold">Find out more about our tasks</a></p>
```

Comment:
A small suggestion: we can remove the `govuk-link` formatting since it's within a Govspeak component and already inherits the link style formatting.

Reply:
There's quite a bit of spacing between the two Govspeak blocks (due to the `govuk-!-margin-bottom-9` spacing). Maybe the text and link could be combined into a single block?

Reply:
You can also override the margin bottom if needed:

https://github.com/alphagov/frontend/blob/7b9dd43935d12345fb87bbcc122a5a04b1a26c3e/app/assets/stylesheets/views/_landing_page.scss#L47-L50

Reply:
Great spots - I've removed `govuk-link` although I've kept it in a separate govspeak block since it looks like the designs call for a bit more spacing between the content and the link. Happy to discuss an alternative approach if preferable though!